api/internal/models/question.go

```diff
@@ -10,6 +10,7 @@ type Question struct {
 	Id        string `json:"id"`
```

Comment:
could you please also adjust the get endpoint for the current session? the voted Property should be returned there as well for every question

Reply:
QuestionDto :)

Reply:
It is already adjusted? (line 366) in brokered or what do you mean?

Reply:
didnt see :)